### PR TITLE
Review all Codeplex links/mentions and update where needed

### PR DESCRIPTION
**Pull request by isaac2004**
Review all Codeplex links/mentions and update where needed #3579

Running PR to port all references to CodePlex to Github/wherever they live now

### CODE REVIEW

File: aspnet/mvc/overview/older-versions-1/nerddinner/index.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 uid: mvc/overview/older-versions-1/nerddinner/index
 title: "NerdDinner | Microsoft Docs"
 author: rick-anderson
-description: "How to build the NerdDinner.com website using ASP.NET MVC 1. For the ASP.MVC 3 version, please visit nerddinner.codeplex.com ."
+description: "How to build the NerdDinner.com website using ASP.NET MVC 1. For the ASP.MVC 3 version, please visit nerddinner on github ."
```
Comment (scottaddie) [inline aspnet/mvc/overview/older-versions-1/nerddinner/index.md]: github --> GitHub

Comment (Rick-Anderson) [reply aspnet/mvc/overview/older-versions-1/nerddinner/index.md]: While you're fixing this remove ~please~

Comment (scottaddie) [reply aspnet/mvc/overview/older-versions-1/nerddinner/index.md]: ASP.MVC 3 --> ASP.NET MVC 3

File: aspnet/mvc/overview/older-versions-1/nerddinner/index.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 uid: mvc/overview/older-versions-1/nerddinner/index
 title: "NerdDinner | Microsoft Docs"
 author: rick-anderson
-description: "How to build the NerdDinner.com website using ASP.NET MVC 1. For the ASP.MVC 3 version, please visit nerddinner.codeplex.com ."
+description: "How to build the NerdDinner.com website using ASP.NET MVC 1. For the ASP.NET MVC 3 version, visit nerddinner on GitHub ."
```
Comment (scottaddie) [inline aspnet/mvc/overview/older-versions-1/nerddinner/index.md]: Also, please remove the extra space before the period at the end of the sentence.

File: aspnet/web-forms/overview/getting-started/hands-on-labs/whats-new-in-web-forms-in-aspnet-45.md [hunk 3]
```diff
@@ -497,7 +497,7 @@ The request validation feature in ASP.NET provides a certain level of default pr
     *Request validation disabled for the product description*
 
     > [!NOTE]
-    > In a production application, you should sanitize the HTML code entered by the user to make sure only safe HTML tags are entered (for example, there are no &lt;script&gt; tags). To do this, you can use [Microsoft Web Protection Library](http://wpl.codeplex.com/).
+    > In a production application, you should sanitize the HTML code entered by the user to make sure only safe HTML tags are entered (for example, there are no &lt;script&gt; tags). To do this, you can use [Microsoft Web Protection Library](https://www.nuget.org/packages/AntiXSS/4.3.0).
```
Comment (scottaddie) [inline aspnet/web-forms/overview/getting-started/hands-on-labs/whats-new-in-web-forms-in-aspnet-45.md]: Let's use the following link instead, so we're not pointing to a specific version: https://www.nuget.org/packages/AntiXSS

File: aspnet/whitepapers/aspnet4/overview.md [hunk 4]
```diff
@@ -1361,7 +1361,7 @@ The following Web sites provide additional information about ASP.NET 4 and Visua
 - [https://www.asp.net/dynamicdata/](https://msdn.microsoft.com/en-us/library/cc488545.aspx) and [ASP.NET Dynamic Data Content Map](https://msdn.microsoft.com/en-us/library/cc488545%28VS.100%29.aspx) — Online resources on the ASP.NET team site and in the official documentation for ASP.NET Dynamic Data.
 - [https://www.asp.net/ajax/](../../ajax/index.md) — The main Web resource for ASP.NET Ajax development.
 - [https://blogs.msdn.com/webdevtools/](https://blogs.msdn.com/webdevtools/) — The Visual Web Developer Team blog, which includes information about features in Visual Studio 2010.
-- [http://www.codeplex.com/aspnet](http://www.codeplex.com/aspnet) — The main Web resource for preview releases of ASP.NET.
+- [ASP.Net WebStack](https://github.com/aspnet/AspNetWebStack) — The main Web resource for preview releases of ASP.NET.
```
Comment (scottaddie) [inline aspnet/whitepapers/aspnet4/overview.md]: ASP.Net --> ASP.NET

File: aspnet/whitepapers/mvc3-release-notes.md [hunk 5]
```diff
@@ -314,7 +314,7 @@ The ASP.NET MVC project templates for Visual Studio were updated to include the
 <a id="RTM-2"></a>
 ### Change: Changed the default ModelMetadataProvider back to DataAnnotationsModelMetadataProvider
 
-The RC2 release of ASP.NET MVC 3 introduced a *CachedDataAnnotationsMetadataProvider* class that provided caching on top of the existing *DataAnnotationsModelMetadataProvider* class as a performance improvement. However, some bugs were reported with this implementation, so the change has been reverted and moved into the MVC Futures project, which is available on the CodePlex website at [http://aspnet.codeplex.com/](http://aspnet.codeplex.com/).
+The RC2 release of ASP.NET MVC 3 introduced a *CachedDataAnnotationsMetadataProvider* class that provided caching on top of the existing *DataAnnotationsModelMetadataProvider* class as a performance improvement. However, some bugs were reported with this implementation, so the change has been reverted and moved into the MVC Futures project, which is available at [ASP.Net WebStack](https://github.com/aspnet/AspNetWebStack).
```
Comment (scottaddie) [inline aspnet/whitepapers/mvc3-release-notes.md]: ASP.Net --> ASP.NET

File: aspnet/whitepapers/what-is-new-in-aspnet-mvc.md [hunk 6]
```diff
@@ -242,7 +242,7 @@ For model objects that use IDataErrorInfo to perform validation, every property
 
 #### IIS script mapping script is no longer available in the installer
 
-The IIS script-mapping script is a command-line script that is used to configure script maps for IIS 6 and for IIS 7 in Classic mode. The script-mapping script is not needed if you use the Visual Studio Development Server or if you use IIS 7 in Integrated mode. The scripts are available as a separate unsupported download on the [ASP.NET CodePlex site](http://aspnet.codeplex.com/).
+The IIS script-mapping script is a command-line script that is used to configure script maps for IIS 6 and for IIS 7 in Classic mode. The script-mapping script is not needed if you use the Visual Studio Development Server or if you use IIS 7 in Integrated mode. The scripts are available as a separate unsupported download on the [ASP.Net WebStack](https://github.com/aspnet/AspNetWebStack).
```
Comment (scottaddie) [inline aspnet/whitepapers/what-is-new-in-aspnet-mvc.md]: ASP.Net --> ASP.NET

**Comment by isaac2004** [comment]
This page references Database Publishing Services which is not on GitHub

https://docs.microsoft.com/en-us/aspnet/web-forms/overview/older-versions-getting-started/deploying-web-site-projects/deploying-a-database-cs

**Comment by isaac2004** [comment]
Tailspin project not on GitHub
https://docs.microsoft.com/en-us/aspnet/web-forms/overview/older-versions-getting-started/tailspin-spyworks/index

**Comment by isaac2004** [comment]
Also references to Issues in CodePlex, how should they be handled?

https://docs.microsoft.com/en-us/aspnet/web-pages/overview/getting-started/introducing-razor-syntax-c

Link above "Variables" section

**Comment by isaac2004** [comment]
Http.sys Manager does not exist on GitHub

http://httpsysmanager.codeplex.com/

**Comment by isaac2004** [comment]
@Rick-Anderson @scottaddie I have commited all references to CodePlex that I can easily do, here are some metrics of remaining references

- 157 References in 40 files to CodePlex

Here are details on references

- 38 References in 11 files to Work Items in CodePlex (mostly WebStack)
- 7 References in 5 files to Release info in CodePlex (Katana as AspNet)
- 97 References in 22 files to Source Controle in Codeplex (readme files, code snippets, etc)

These are references that only exist in CodePlex, let me know next steps.

File: aspnet/whitepapers/mvc3-release-notes.md [hunk 7]
```diff
@@ -628,13 +628,13 @@ ASP.NET MVC 3 Beta was released on October 6, 2010. The following notes are spec
...
+ASP.NET MVC 3 includes NuGet Package Manager, which is an integrated package management tool for adding libraries and tools to Visual Studio projects. For the most part, it automates the steps that developers take today to get a library into their source tree.
 
-You can work with NuPack as a command line tool, as an integrated console window inside Visual Studio 2010, from the Visual Studio context menu, and as set of PowerShell cmdlets.
+You can work with NuGet as a command line tool, as an integrated console window inside Visual Studio 2010, from the Visual Studio context menu, and as set of PowerShell cmdlets.
 
-For more information about NuPack, visit [http://nupack.codeplex.com/](http://nupack.codeplex.com/) and read the [Getting Started Guide](http://nupack.codeplex.com/documentation?title=Getting%20Started).
+For more information about Nuget, read the [NuGet Documentation](https://docs.microsoft.com/en-us/nuget/).
```
Comment (scottaddie) [inline aspnet/whitepapers/mvc3-release-notes.md]: - Nuget --> NuGet
- remove "en-us/" from the URL

**Comment by Rick-Anderson** [comment]
> Also references to Issues in CodePlex, how should they be handled?

Link above [Variables](https://docs.microsoft.com/en-us/aspnet/web-pages/overview/getting-started/introducing-razor-syntax-c#variables)

Razor has limitations on nested blocks of C#. For more information see [Named C# Variables and Nested Blocks Generate Broken Code](http://aspnetwebstack.codeplex.com/workitem/1914)

Leave those for now.

**Comment by Rick-Anderson** [comment]
>This page references Database Publishing Services which is not on GitHub

Even better, if your web host provider supports Microsoft s [Database Publishing Services](http://www.codeplex.com/sqlhost/Wiki/View.aspx?title=Database%20Publishing%20Services&referringTitle=Home)  you can have the script generated by the Database Publishing Wizard automatically executed on the database server on your behalf. 

Leave that for now. CodePlex won't go away, it's just read-only.

**Comment by Rick-Anderson** [comment]
> Http.sys Manager does not exist on GitHub

http://httpsysmanager.codeplex.com/

Can't use use the RO codeplex link?

**Comment by isaac2004** [comment]
@Rick-Anderson I won't touch the issues references, what about Release Info and Code Snippets?

**Comment by Rick-Anderson** [comment]
@isaac2004   these?

- 7 References in 5 files to Release info in CodePlex (Katana as AspNet)
- 97 References in 22 files to Source Controle in Codeplex (readme files, code snippets, etc)

Can you provide links to specific examples? Are there references in [An Overview of Project Katana](https://docs.microsoft.com/en-us/aspnet/aspnet/overview/owin-and-katana/an-overview-of-project-katana) that need to be fixed?

**Comment by isaac2004** [comment]
@Rick-Anderson 

Examples of Release References

- [Halfway down page](https://docs.microsoft.com/en-us/aspnet/mvc/overview/older-versions-1/deployment/using-asp-net-mvc-with-different-versions-of-iis-vb)
- [NuGet 2.8.1 Section](https://docs.microsoft.com/en-us/aspnet/visual-studio/overview/2013/aspnet-and-web-tools-20132-preview-for-visual-studio-2013-release-notes)

Examples of Source Code References
- [All Samples here](https://docs.microsoft.com/en-us/aspnet/aspnet/overview/owin-and-katana/katana-samples)
- [Multiple Code Samples Here](https://docs.microsoft.com/en-us/aspnet/identity/overview/extensibility/implementing-a-custom-mysql-aspnet-identity-storage-provider)

I think it is probably safe to leave these here as they are just references to code, as long as the code never gets removed from CodePlex

**Comment by Rick-Anderson** [comment]
@isaac2004  awesome work. Thanks for doing this.

**Pull request merged**